File: models/device/device.go
package devicemodels

type Device struct {
	CreationDate *int64 `bson:"creationdate,omitempty"`
	LastUpdate   *int64 `bson:"updatedate,omitempty"`
	User         string `bson:"user,omitempty"`
	Address      string `bson:"address,omitempty"`
	UserAgent    string `bson:"useragent,omitempty"`
	Token        string `bson:"token,omitempty"`
}

File: src/main/kotlin/org/valhalla/core/sdk/model/device/Device.kt
package org.valhalla.core.sdk.model.device

import kotlinx.serialization.Serializable

/**
 * This class represents a user device logged in the platform
 * and the corresponding token.
 */
@Suppress("unused")
@Serializable
data class Device(
    var address: String? = null,
    var userAgent: String? = null,
    var token: String? = null
)